src/ipLoadBalancing/farm/ipLoadBalancing-farm-server.service.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+_.forEach(["tcp", "udp", "http"], function (type) {
```

Comment:
I don't put comment on this file, but my previous comment can be applied ;)

Reply:
done